### PR TITLE
Visual tools for CI/CD pipeline definition

### PR DESCRIPTION
**Pull request by vittoriop17**
# Assignment Proposal

## Title

Visual tools for CI/CD pipeline definition

## Names and KTH ID

-   Vittorio Pellegrini (vpel@kth.se)
-   Mayuri Salunke (salunke@kth.se)

## Deadline

Task 1

## Category

Essay

## Description

Nowadays, CI/CD pipelines can be defined by exploiting several tools. Each tool has its grammar and capabilities.
Usually, such tools require the definition of ad-hoc configuration files (YAML files) with specific keywords and values.
The complexity resides in the definition of such configuration, which is time-consuming and hard to debug.
Visual tools should simplify the definition of the overall configuration.

In conclusion, I would also like to propose some ideas. These could represent future aspects to develop to improve CI/CD definition, regardless of the tool.

### CODE REVIEW

**Comment by khesoem** [comment]
@vittoriop17 Your pull request is adding two readme files. Please remove one of them (I guess the one with an unregistered student should be removed)
https://github.com/KTH/devops-course/pull/1663/files

**Comment by khesoem** [comment]
As we have already passed Task 1 deadline, I'm closing this PR.
